addon/bootstrap.js

```diff
@@ -11,4 +15,9 @@ function shutdown(data, reason) {}
 function install(data, reason) {}
 function uninstall(data, reason) {}
 
-function handleMessage() {}
+function handleMessage(msg, sender, sendReply) {
+  if (msg && msg.funcName === "getTelemetryPref") {
+    let enableTelemetry = prefs.getBoolPref("toolkit.telemetry.enabled");
```

Comment:
If you're willing to limit the add-on to 54 and above, then [bug 1338306](https://bugzilla.mozilla.org/show_bug.cgi?id=1338306) just landed, which lets you supply an optional argument which will avoid the throw.